### PR TITLE
Don't use knowit quality to check for quality in history

### PR DESCRIPTION
**Pull request by medariox**
As the snatched quality has to come from guessit. Avoid wrong matches in history and allows matches with unknown quality.

@fernandog @OmgImAlexis 

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by fernandog** [comment]
@OmgImAlexis can you test this please?

File: medusa/post_processor.py [hunk 1]
```diff
@@ -731,26 +731,33 @@ def _get_ep_obj(self, show, season, episodes):
...
+        """
+        Determine the quality of the file that is being post processed with its status.
+
+        :param status: The status related to the file we are post processing
+        :return: A quality value found in common.Quality
         """
-        Determine the quality of the file that is being post processed.
+        quality = common.Quality.UNKNOWN
+
+        if status in common.Quality.SNATCHED + common.Quality.SNATCHED_PROPER + common.Quality.SNATCHED_BEST:
```
Comment (medariox) [inline medusa/post_processor.py]: This should probably be removed, as it just assigns the quality of the current snatched status to the file that is being processed.

Comment (medariox) [inline medusa/post_processor.py]: Or does anyone have an idea why something like that should be done?

**Comment by OmgImAlexis** [comment]
Will test in a bit.

**Comment by OmgImAlexis** [comment]
@medariox still can't test as I'm having issues with symlinks and stuff that @fernandog is working on.

**Comment by fernandog** [comment]
@OmgImAlexis worked then?

**Comment by OmgImAlexis** [comment]
As far as I can tell.

**Pull request merged**